.eslintrc.json

```diff
@@ -1,3 +1,18 @@
 {
-    "extends": ["@apify"]
-}
+    "extends": [
+      "@apify/ts"
+    ],
+    "parserOptions": {
+      "project": "tsconfig.eslint.json"
+    },
+    "overrides": [
+      {
+        "files": [
+          "test/**/*.js"
```

Comment:
The tests are very JS-ly complicated and I believe it would take more time than the source code itself.

Reply:
the motivation for me would be to run tests directly on source code without compilation - i believe that should be possible even if we keep the tests in JS and use ts-node on them (and importing stuff from `src` instead of `dist`)

wdyt about that? definitely not a priority to rewrite it if it would be too time consuming

Reply:
That sounds very cool, I'll try this :)

Reply:
Fixed! :D